src/App.js

```diff
@@ -1,8 +1,13 @@
 import { Component } from 'react';
-import Calculator from './Calculator';
+import Calculator from './components/Calculator';
 
```

Comment:
- [ ] Your calculator app is not functioning as intended. Running the operation 9/0 yields a blank screen as below. Kindly ensure to improve upon this aspect.

**Operation Error:**
![image](https://user-images.githubusercontent.com/12745474/158649441-9d61aa9e-a952-4b09-a796-e046e39ac352.png)